packages/core/admin/admin/src/pages/MarketplacePage/hooks/use-navigator-on-line.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+
+const useNavigatorOnLine = () => {
```

Comment:
I did grab it from somewhere. That's a cool site but I didn't get it from there, I found it [here](https://www.30secondsofcode.org/react/s/use-navigator-on-line) and thought it looked good. Should I credit them?

Ok I hesitated to put it there but since for now this was the only place it's being used I put it here. I'll move it.

Reply:
I think it would be nice to link that page in a comment, it adds a bit more context and explains how it works